### PR TITLE
Modified multipath conf for SLES

### PR DESCRIPTION
**Pull request by bhagyashree-sarawate**
Hi Sneha,
As per our discussion i have added the code for SLES modifications.
Please Review.
PFA installer logs

Bhagyashree Sarawate
[installer_logs.txt](https://github.com/hpe-storage/python-hpedockerplugin/files/3201732/installer_logs.txt)

### CODE REVIEW

**Comment by bhagyashree-sarawate** [comment]
Attaching test logs using two nodes using standalone driver
[installer_logs_using_2_nodes.txt](https://github.com/hpe-storage/python-hpedockerplugin/files/3206950/installer_logs_using_2_nodes.txt)

**Comment by sneharai4** [comment]
One minor review comment for task file install_prerequisites_on_all.yml. I see " [WARNING]: conditional statements should not include jinja2 templating delimiters such as {{ }} or {% %}. Found: "{{item}}" not in default_drivers" in both the logs attached.  Please fix this warning.

File: ansible_3par_docker_plugin/tasks/install_prerequisites_on_all.yml [hunk 1]
```diff
@@ -31,8 +31,7 @@
       msg: "Driver {{item}} is incorrect"
     with_items:
       - "{{my_driver_list}}"
-    when: >-
-        "{{item}}" not in default_drivers       
+    when: item not in default_drivers
```
Comment (sneharai4) [inline ansible_3par_docker_plugin/tasks/install_prerequisites_on_all.yml]: @bhagyashree-sarawate Remove white-spaces at the end.

**Comment by sneharai4** [comment]
Adding logs 
[test_logs.txt](https://github.com/hpe-storage/python-hpedockerplugin/files/3223159/test_logs.txt)

**Pull request merged**